configmap/example.go

```diff
@@ -30,9 +31,14 @@ const (
 	ExampleChecksumAnnotation = "knative.dev/example-checksum"
 )
 
+var (
+	// Allows for normalizing by collapsing newlines.
+	sequentialNewlines = regexp.MustCompile("\n+")
```

Comment:
ugh, what about `\r\n` :) you know windows users

Reply:
Sure, we could do that. I thought I had seen hard coded path separators somewhere... We support windows?

Reply:
well you edit CM in the client env?

Reply:
Ah, fair enough. And you're right, the window kubectl defaults to the system default:

https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands